src/Robo/Plugin/Commands/DeployCommands.php

```diff
@@ -73,5 +84,47 @@ public function deployDrupal(string $appDirPath, string $siteName = 'default', s
             // https://github.com/drush-ops/drush/issues/2449#issuecomment-708655673
             ->exec("$appDirPath/vendor/bin/drush config:import --yes")
```

Comment:
I am typically in the camp of being as explicit as possible in scripts or code like this so that we aren't ever relying on trusting another system to be configured a certain way ($PATH in this case). I suppose I figure that those characters are free!

Reply:
> I am typically in the camp of being as explicit as possible in scripts or code like this so that we aren't ever relying on trusting another system to be configured a certain way ($PATH in this case). I suppose I figure that those characters are free!

🏑 fight! 😆

Reply:
I missed this before, but let's call this a refactor for another day.